src/test/java/org/takes/http/BkParallelTest.java

```diff
@@ -80,12 +83,16 @@ public Response act(final Request req)
             }
         };
         new Thread(
+            // @checkstyle AnonInnerLengthCheck (23 lines)
             new Runnable() {
                 @Override
                 public void run() {
                     try {
                         new FtBasic(
-                            new BkParallel(new BkBasic(take)),
+                            new BkParallel(
+                                new BkBasic(take),
+                                count
```

Comment:
@yegor256 in some cases there is not 3 available threads and I think that
the problem. Two threads are blocked and there isn't any more to count down
till 0 and release the others
El 29/05/2015 21:29, "Yegor Bugayenko" <notifications@github.com> escribió:

> In src/test/java/org/takes/http/BkParallelTest.java
> <https://github.com/yegor256/takes/pull/316#discussion_r31375028>:
>
> >              new Runnable() {
> >                  @Override
> >                  public void run() {
> >                      try {
> >                          new FtBasic(
> > -                            new BkParallel(new BkBasic(take)),
> > +                            new BkParallel(
> > +                                new BkBasic(take),
> > +                                count
>
> @lautarobock <https://github.com/lautarobock> but you will always have 1
> or 2 or 3 threads, not more. now we always have 3. how do 1 or 2 solve the
> problem?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yegor256/takes/pull/316/files#r31375028>.
>